src/Compilers/CSharp/Test/Symbol/Symbols/Source/BaseClassTests.cs

```diff
@@ -2070,8 +2070,11 @@ AbstractClass Test()
 }";
             var comp = CreateCompilation(text);
             comp.VerifyDiagnostics(
+    // (2,14): error CS0311: The type 'object' cannot be used as type parameter 'T' in the generic type or method 'Crash<T>'. There is no implicit reference conversion from 'object' to 'CrashTest.Crash<object>.AbstractClass'.
+    // using static CrashTest.Crash<object>;
+    Diagnostic(ErrorCode.ERR_GenericConstraintNotSatisfiedRefType, "CrashTest.Crash<object>").WithArguments("CrashTest.Crash<T>", "CrashTest.Crash<object>.AbstractClass", "T", "object").WithLocation(2, 14),
```

Comment:
This test contains a using static which violates a generic constraint. Before, we emitted a diagnostic on the usage of a class contained within the type imported via `using static`, but it seems correct to add a diagnostic to the `using static` directive itself here too.